workflows/ascc_genomic.nf

```diff
@@ -111,13 +87,14 @@ workflow ASCC_GENOMIC {
     } else {
         log.warn("MAKE SURE YOU ARE AWARE YOU ARE SKIPPING ESSENTIAL JOBS, THIS INCLUDES BREAKING SCAFFOLDS OVER 1.9GB, FILTERING N\'s AND GC CONTENT REPORT (THIS WILL BREAK OTHER PROCESSES AND SHOULD ONLY BE RUN WITH `--include essentials`)")
 
-        reference_tuple_from_GG = ch_samplesheet // This is the reference genome input channel
+        reference_tuple_from_GG = ch_samplesheet
+        ej_dot_genome           = Channel.of([[],[]])
```

Comment:
Should these be `Channel.empty()`? `[[],[]]` is a valid channel input so processes relying on essentials will fail with that input - whereas an empty channel will simply not start the process as a process doesn't start until all input slots are filled.

Reply:
It's an artifact of pre-ascc-genomic.nf lines (545-571). I needed to pass in all the channels one at a time, but for each run there could be any number of channels actually being populated and the processes would still need to run.

The above code chunk fixed that and I just never fixed up the empties.

Done now.